agent/apiharness/src/main/java/com/intuit/tank/harness/APITestHarness.java

```diff
@@ -381,10 +381,11 @@ private void saveDataFile(DataFileRequest dataFileRequest) {
                         "writing file " + dataFileRequest.getFileName() + " to " + dataFile.getAbsolutePath()
                                 + " from url " + url.toExternalForm())));
                 FileUtils.copyURLToFile(url, dataFile);
-                if (dataFileRequest.isDefault()
+                if (dataFileRequest.isDefaultDataFile()
```

Comment:
Oh, it confused me as well. The workflow is that it checks if there's a single datafile (in JobManager), and if there is, sets the conditional `isDefaultDataFile` to `true` from the controller. It is then sent to each agent which checks (in APITestHarness) whether both the default data flag is `true` and makes sure that it doesn't share a name with the default file name `csv-data.txt`. That's because it then copies the content from the original file to `csv-data.txt` which is then referenced throughout the rest of the code as the default datafile. I do agree that we should rename this, and maybe even revisit the workflow itself to just use the original file.